opslevel/resource_opslevel_check_repository_grep.go

```diff
@@ -108,7 +108,7 @@ func (r *CheckRepositoryGrepResource) Schema(ctx context.Context, req resource.S
 func (r *CheckRepositoryGrepResource) ValidateConfig(ctx context.Context, req resource.ValidateConfigRequest, resp *resource.ValidateConfigResponse) {
 	var configModel CheckRepositoryGrepResourceModel
 	resp.Diagnostics.Append(req.Config.Get(ctx, &configModel)...)
-	if resp.Diagnostics.HasError() {
+	if resp.Diagnostics.HasError() || configModel.FileContentsPredicate.Type.IsUnknown() {
```

Comment:
What about the `== nil` check here?

Reply:
This nil check is not need here since [FileContentsPredicate](https://github.com/OpsLevel/terraform-provider-opslevel/pull/361/files/8049c8a5b822f7f3e30bb576834d018c7e386b7e#diff-9c818595e0e2fc40416a502fc1a0e6fa4b747a871c57c12f7e86870ff42b0369R46) is not a pointer in `opslevel_check_repository_grep` and in `opslevel_check_repository_search`

Reply:
@davidbloss got it, I was looking at an old version of `main` I had locally.